packages/website/pages/docs/how-to/get-status.md

```diff
@@ -0,0 +1,82 @@
+
+If your data was pinned using the Pinning Service API, a `Failed` status usually means that the cluster wasn't able to fetch the complete content over the IPFS peer-to-peer network. These issues can be hard to diagnose due to the many variables involved, especially when original source of the data may have an unreliable connection to the network or is behind a firewall. If the data is being provided by another pinning service such as Pinata, or if you've verified that the data is retrievable using IPFS (ideally from multiple distinct locations), please [file an issue][new-issue] or join the `#nft-storage` channel on the [IPFS Discord][support-discord] so we can dig into the cause.
+
+Please note that the status reported by the NFT.Storage service may not always be in sync with the actual status as reported by the IPFS cluster. The NFT.Storage service checks if `Failed` pins have been found and pinned by the cluster once per week for up to one month after the initial upload. 
+
+If you find that your file is retrievable but the pin status still says `Failed`, please [file an issue][new-issue] with the relevant CIDs and we'll take a look. If you'd prefer to keep the CIDs private, reach out to one of the devs on the `#nft-storage` [Discord channel][support-discord] in a direct message.
+
+## Storage providers
+
+The **Storage Providers** column on the file listing page shows information about the [Filecoin storage providers][fil-docs-storage-providers] that have committed to long-term storage deals for the uploaded data.
+
+When data is first uploaded, the Storage Providers column will show a status of `Queuing`. This indicates that the NFT.Storage service has the data and is aggregating it into a format that can be used to make a storage deal.
+
+Once enough data has been aggregated, the NFT.Storage service will propose several Filecoin [storage deals][fil-docs-deals] with different Filecoin storage providers. These will initially be shown in a "pending" status, while the provider accepts the deal and posts its commitment to the Filecoin blockchain.
+
+When using the HTTP API or client library to check the upload status, information about Filecoin storage is provided in the `deals` array field within the status response object. The `deals` array will be empty while the upload is in the `Queuing` state. 
+
+Once storage deals have been proposed, the `deals` array will contain JSON objects describing each deal. Each object has a `status` field, which will have a value of `"queued"` when the deal is pending. When the deal has been confirmed and published to the Filecoin blockchain, the `status` field will have a value of `"active"`.
+
+Here's an example JSON object that describes an active storage deal:
```

Comment:
Would also add: "You can verify the status of your upload itself on the Filecoin blockchain at any time after the initial storage deals are made."